ProjectStructureAndRoles.md

```diff
@@ -2,42 +2,95 @@
+
+**API Repositories**
+
+The purpose of an API Repository is the definition and documentation of one or multiple (very closely related) APIs which will be always released together.
 
-A Sub Project should have at least one Maintainer. Ideally a Sub Project is managed by two or more Maintainers, depending on the size and scope of the Sub Project. The Maintainers of a Sub Project are defined by the union of the Maintainers listed within the MAINTAINERS.md files of the API Repositories which are belonging to the Sub Project. It is recommended that there is an overlap between the Maintainers of the API Repositories of the Sub Project, but it is not mandated that all Maintainers of the Sub Project take responsibility for all APIs in the Sub Project.
+A repository must have at least one Codeowner. Repositories with branch protection rules must have at least two Codeowners. Only Codeowners should have write permissions to an API Repository. For requirements regarding the Maintainers see the definition of Sandbox, Incubated and Graduated API Repositories.
+
+Each API Repository must have a README.MD file (with a description of the API scope), a license file, a GOVERNANCE.MD file (pointing to the Governance repository), a CODEOWNERS, and a MAINTAINERS.MD file.
+
+Each API Repository should have subdirectories /documentation/API_documentation, /documentation/SupportingDocuments, /code/API_definitions, and /code/Test_definitions. 
   
-A Provider Implementation Repository only must have a CODEOWNERS file, a license file, a GOVERNANCE.MD file (pointing to the Governance repository) and a subdirectory /code/API_code with API code of the transformation function Northbound --> Southbound. Only Codeowners should have write permissions to a Provider Implementation (PI) Repository. Provider Implementation Repositories have the same name as the API Repository but with suffix _-PI§x§_.
+API Repositories can have in addition Provider Implementation (PI) Repositories on their side. Provider Implementation Repositories only must have a CODEOWNERS file, a license file, a GOVERNANCE.MD file (pointing to the Governance repository) and a subdirectory /code/API_code with API code of the transformation function Northbound --> Southbound. Only Codeowners should have write permissions to a Provider Implementation (PI) Repository. Provider Implementation Repositories have the same name as the API Repository but with suffix _-PI§x§_.
 
-Where a Sub Project has a release process, access and documentation should be such that more than one person can perform a release.
+**Sandbox API Repositories**
 
-**Working Groups**
+A Sandbox API Repository is meant for the rapid development of a new API proposal through one or multiple initial API versions (0.x.y) until the first public release of an initial or stable API Version within the context of a CAMARA Meta-release.
 
```

Comment:
Even when sandbox proposals are not detaily analized that will not occur until they have progressed and have a clearer scope and technical definition, We need to include basic rules that ensures that proposals included as sandbox, are still following the spirit of CAMARA.
```suggestion
Sandbox API Repositories 
```

Reply:
I agree with the intention of the suggested change, but would avoid to rephrase the scope (defined in the ProjectCharter) or technical API design guidelines in this document. Instead I added with 3b49da2449854875a7f8419f9fc3b58af6f1d1ce the following sentence:

> The work within Sandbox API Repositories - like in all API repositories - shall follow the governance and guidelines of CAMARA, including but not limited to the ProjectCharter, the Code of Conduct, the Contribution guidelines, the API Design Guidelines of Commonalities and the Release Process defined in Release Management.

Reply:
Agreed, ok.